static/js/reducers/course_prices.js

```diff
@@ -21,22 +21,37 @@ import type {
 import { updateStateByUsername } from './util';
 
 export const INITIAL_COURSE_PRICES_STATE: CoursePricesState = {
-  coursePrices: []
+  coursePrices: [],
+  noSpinner:    false
 };
 
 export const prices = (state: CoursePriceReducerState = {}, action: Action<any, string>) => {
   const { meta: username } = action;
   switch (action.type) {
   case REQUEST_COURSE_PRICES:
-    return updateStateByUsername(
-      R.dissoc(username, state),
-      username,
-      _.merge({}, INITIAL_COURSE_PRICES_STATE, { fetchStatus: FETCH_PROCESSING })
-    );
+    if (action.payload === true) {
```

Comment:
Please remove this if statement. Since the only difference between the two cases is the `noSpinner`, you can have this instead:

    return updateStateByUsername(
      R.dissoc(username, state),
      username,
      _.merge({}, INITIAL_COURSE_PRICES_STATE, {
         noSpinner: action.payload,
        fetchStatus: FETCH_PROCESSING
      })
    );

Please do the same for the dashboard reducer